### ISSUE TITLE
Map Objects returning empty data

### ISSUE THREAD

**Issue by MikeShiner** [cyraxx/pogobuf#116]
For some reason I can log in fine with the API but after calling getMapObjects, each mapObject returned is empty. I'm getting:

```
Cell 5220699518646878208
Has 0 catchable Pokemon
{ s2_cell_id: '5220699520794361856',
  current_timestamp_ms: 1482803436266,
  forts: [],
  spawn_points: [],
  deleted_objects: [],
  is_truncated_list: false,
  fort_summaries: [],
  decimated_spawn_points: [],
  wild_pokemons: [],
  catchable_pokemons: [],
  nearby_pokemons: [] }
```
This has happened on many many different coordinates across many accounts. Is this an issue with the API itself?

**Comment by joelfi** [comment]
Due to constant changes in Pokémon GO API, `node-pogo-signature` hasn't been updated yet to make valid signatures for latest supported API. I have heard, that only one group has "cracked" the new signature and there aren't others cracking it. This one group is selling access to their paid signature making API with "requests per minute"-limits (more money = more requests). `node-pogo-signature` is most likely updated to support this paid service in the future (if I have interpreted the discussions in /r/pokemongodev Discord correctly) so you could just put in your paid API token and start using the API. I'm labelling this as a bug for now.

**Comment by nicoschmitt** [comment]
Use this one in the meantime: https://github.com/pogosandbox/node-pogo-signature.
It come from pcrypt beeing update for 0.49 but hashing still on 0.45.

**Comment by joelfi** [comment]
Fixed with 0780e2e6e09dd54a23f5933d89f6544c83bde893
